plugins/chat/spec/system/post_send_message_warnings_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-RSpec.describe "JIT messages", type: :system, js: true do
+RSpec.describe "Post-send message warnings for mentions", type: :system, js: true do
```

Comment:
Lots of flakiness here. Locally, warnings comes up instantly, so it's not clear what's going on.